src/client/components/household-information/FinancialDisclosureSection.jsx

```diff
@@ -73,8 +74,8 @@ class FinancialDisclosureSection extends React.Component {
                 errorMessage={isValidFinancialDisclosure(this.props.data) ? '' : 'Please acknowledge this requirement'}
                 label="I understand VA is not currently enrolling new applicants who decline to provide their financial information unless they have other qualifying eligibility factors."
                 name="understandsFinancialDisclosure"
-                checked={this.props.data.understandsFinancialDisclosure}
-                onValueChange={(update) => {this.props.onStateChange('understandsFinancialDisclosure', update);}}/>
+                checked={this.props.data.understandsFinancialDisclosure.value}
+                onValueChange={(update) => {this.props.onStateChange('understandsFinancialDisclosure', { value: update, dirty: false });}}/>
```

Comment:
the checkbox only sends the value and not the field with the dirty flag (this is different from the other field inputs), so with just update we would lose the dirty flag on the field